src/Hooks/LogHandler.php

```diff
@@ -27,7 +27,7 @@ public function __construct(
 
     public function isHandling(LogRecord $record): bool
     {
-        return $this->nightwatch->shouldCaptureLogs() && $this->level->includes($record->level);
+        return $this->nightwatch->enabled() && $this->nightwatch->shouldCaptureLogs() && $this->level->includes($record->level);
```

Comment:
This shouldn't be required, as `shouldCaptureLogs` already checks for `enabled`.